### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by PriyadarshiS** [community/community#114400] [GitHub Education]
### Select Topic Area

Question

### Body

I applied for student developer pack 6 days ago but still it says we' re still processing your application. When will I get verified

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @PriyadarshiS! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
